src/main/java/io/github/thebusybiscuit/slimefun4/implementation/resources/OrganicNetherGoo.java

```diff
@@ -0,0 +1,36 @@
+package io.github.thebusybiscuit.slimefun4.implementation.resources;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.core.attributes.PiglinBarterDrop;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * This {@link SlimefunItem} can only be obtained via bartering with a {@link Piglin}. 
```

Comment:
This description does no cover anything about what this class (Organic Goo) is.
It is too generic.